ngraph/test/backend/tan.in.cpp

```diff
@@ -62,3 +62,24 @@ NGRAPH_TEST(${BACKEND_NAME}, tan)
                                                 -1.15782128f},
                                   read_vector<float>(result)));
 }
+
+
+NGRAPH_TEST(${BACKEND_NAME}, tan_int32)
+{
+    Shape shape{5};
+    auto A = make_shared<op::Parameter>(element::i32, shape);
+    auto f = make_shared<Function>(make_shared<op::Tan>(A), ParameterVector{A});
+
+    auto backend = runtime::Backend::create("${BACKEND_NAME}");
+
+    // Create some tensors for input/output
+    auto a = backend->create_tensor(element::i32, shape);
+    vector<int32_t> input{-2, -1, 0, 1, 2};
+    copy_data(a, input);
+    auto result = backend->create_tensor(element::i32, shape);
+
+    auto handle = backend->compile(f);
+    handle->call_with_validate({result}, {a});
+    vector<int32_t> expected{2, -2, 0, 2, -2};
```

Comment:
how did you generate these reference numbers? Are they match to frameworks' ones. I just tried the latest TF and it doesn't support int32 for tan

Reply:
hi rkazants, I generated the expected numbers by calculator. After got your question,  I installed TF and found it's true that TF don't support int32, conflict with it's document [tan](https://www.tensorflow.org/api_docs/python/tf/math/tan).
I write this patch refer to [6249](https://github.com/openvinotoolkit/openvino/pull/6429) and [6288]int(https://github.com/openvinotoolkit/openvino/pull/6288), and they all added int32 test case

Reply:
With PyTorch it seems ok.